src/libpriv/rpmostree-core.cxx

```diff
@@ -154,10 +154,8 @@ set_rpm_macro_define (const char *key, const char *value)
 RpmOstreeContext *
 rpmostree_context_new_base (OstreeRepo   *repo)
 {
-  g_assert (repo != NULL);
```

Comment:
This part is going to be...interesting.  Heavy intersection with https://github.com/ostreedev/ostree-rs-ext/issues/159